data/xml/Q18.xml

```diff
@@ -493,6 +496,7 @@
       <abstract>Can advances in NLP help advance cognitive modeling? We examine the role of artificial neural networks, the current state of the art in many common NLP tasks, by returning to a classic case study. In 1986, Rumelhart and McClelland famously introduced a neural architecture that learned to transduce English verb stems to their past tense forms. Shortly thereafter in 1988, Pinker and Prince presented a comprehensive rebuttal of many of Rumelhart and McClelland’s claims. Much of the force of their attack centered on the empirical inadequacy of the Rumelhart and McClelland model. Today, however, that model is severely outmoded. We show that the Encoder-Decoder network architectures used in modern NLP systems obviate most of Pinker and Prince’s criticisms without requiring any simplification of the past tense mapping problem. We suggest that the empirical performance of modern networks warrants a reexamination of their utility in linguistic and cognitive modeling.</abstract>
       <pages>651–665</pages>
       <url>Q18-1045</url>
+      <video href="https://vimeo.com/305684335" tag="video"/>
     </paper>
     <paper id="46">
```

Comment:
This one’s good but I want to comment that I hate how many of the videos have title-final periods.